tool/tctl/common/token_command.go

```diff
@@ -113,7 +113,7 @@ func (c *TokensCommand) Initialize(app *kingpin.Application, config *servicecfg.
 
 	// tctl tokens add ..."
 	c.tokenAdd = tokens.Command("add", "Create a invitation token.")
-	c.tokenAdd.Flag("type", "Type(s) of token to add, e.g. --type=node,app,db").Required().StringVar(&c.tokenType)
+	c.tokenAdd.Flag("type", "Type(s) of token to add, e.g. --type=node,app,db,proxy,etc").Required().StringVar(&c.tokenType)
```

Comment:
Would it make sense to list all token types here instead of using `etc`?

Reply:
I have two concerns about that:

1. The additional content will add to the "wall of text" and make the overall help output harder to parse.
2. Developers won't consistently update this text when token types are added or changed, as evidenced by the current state.

Reply:
I'm not as big a fan of "e.g" as you are. I worry that it may confuse some who aren't familiar with it as it's not used often outside of technical and academic writing. Doubley so for ESL readers.

That said, similar arguments could be made for "etc" or "et cetera". @lsgunn-teleport would you care to weigh in?

Reply:
I would recommend we always use common English (translatable) words instead of Latin or Latin abbreviations. Not everyone knows the intended meaning for e.g. or i.e. (in particular).
e.g. > use "for example"
i.e. > use "that is" (or rewrite so it isn't necessary to add this qualifier)
etc. > use "and so on" (or rewrite so it isn't necessary)
via > use an appropriate and less ambiguous word (though? by? from? using? by way of? by means of?)

Reply:
How about:

```suggestion
	c.tokenAdd.Flag("type", "Type(s) of token to add, such as --type=node,app,db,proxy,etc").Required().StringVar(&c.tokenType)
```